### ISSUE TITLE
Where is the Fortran Lang governance set out?

### ISSUE THREAD

**Issue by logankilpatrick** [fortran-lang/webpage#211]
I was looking through the website and can't seem to find the Fortran Lang governance model documented anywhere. Can someone point me too it?

**Comment by logankilpatrick** [comment]
I see this https://github.com/fortran-lang/.github/blob/main/GOVERNANCE.md but it is not linked anywhere on the website.

**Comment by awvwgk** [comment]
Good point we should move it to the community section on the webpage. This way it can also be localized.

**Comment by Moksh45** [comment]
Hello,
I would like to work on this issue. Could you please assign it to me?

**Comment by awvwgk** [comment]
@Moksh45 feel free to send a patch. Let us know if you need help with sphinx or MyST.

**Comment by Moksh45** [comment]
ok

**Comment by Moksh45** [comment]
> @Moksh45 feel free to send a patch. Let us know if you need help with sphinx or MyST.

ok

**Comment by milancurcic** [comment]
It may be too large for the entire content to be included on the community page. Perhaps it can its own page, say, https://fortran-lang.org/en/governance/, but link to it from a section in the community page? The link could also be an item in the navbar.

**Comment by awvwgk** [comment]
I would put it under community/governance

**Comment by Moksh45** [comment]
@milancurcic @awvwgk I will do my utmost to implement this and incorporate your suggestions.

**Comment by Moksh45** [comment]
@awvwgk @milancurcic I'm having difficulty with something 
![Screenshot from 2023-01-19 10-23-31](https://user-images.githubusercontent.com/92110858/213248930-0c1d32b7-d90d-4777-89af-e15350722bbf.png)

What is the best way to remove the search box?

**Comment by Moksh45** [comment]
@milancurcic @awvwgk Could you kindly take a look at my pull request and consider merging it?
